letta/client/streaming.py

```diff
@@ -6,7 +6,7 @@
 
 from letta.constants import OPENAI_CONTEXT_WINDOW_ERROR_SUBSTRING
 from letta.errors import LLMError
-from letta.schemas.enums import MessageStreamStatus
+from letta.llm_api.openai import OPENAI_SSE_DONE
```

Comment:
I'm using the constant for the equality checks and my new literal type for the type hint on the union used for the rv